### PR TITLE
Plasmaman Suit Enhancement

### PR DESCRIPTION
**Pull request by SkowronX**
*Gives names and descriptions.
*New suit sprites for head roles + other jobs
*Plasmamen nukeops no longer spawn without suit and internals
*Plasma tanks no longer start out at 101kpa release.

plasmamansuits

### CODE REVIEW

File: code/modules/clothing/spacesuits/plasmamen.dm [hunk 1]
```diff
@@ -1,6 +1,8 @@
 // PLASMEN SHIT
 // CAN'T WEAR UNLESS YOU'RE A PINK SKELLINGTON
 /obj/item/clothing/suit/space/plasmaman
+	name = "plasmaman suit"
+	desc = "A special containment suit designed to protect a plasmaman's volatile body from outside exposure and quicly extinguish it in emergencies."
```
Comment (BestName) [inline code/modules/clothing/spacesuits/plasmamen.dm]: >quicly

File: code/game/gamemodes/nuclear/nuclear.dm [hunk 2]
```diff
@@ -212,9 +212,17 @@
 
 	synd_mob.equip_to_slot_or_del(new /obj/item/clothing/under/syndicate(synd_mob), slot_w_uniform)
 	synd_mob.equip_to_slot_or_del(new /obj/item/clothing/shoes/combat(synd_mob), slot_shoes)
-	synd_mob.equip_to_slot_or_del(new /obj/item/clothing/suit/armor/bulletproof(synd_mob), slot_wear_suit)
+	if(!istype(synd_mob.species, /datum/species/plasmaman))
+		synd_mob.equip_to_slot_or_del(new /obj/item/clothing/suit/armor/bulletproof(synd_mob), slot_wear_suit)
+	else
+		synd_mob.equip_to_slot_or_del(new /obj/item/clothing/suit/space/plasmaman/nuclear(synd_mob), slot_wear_suit)
+		synd_mob.equip_to_slot_or_del(new /obj/item/weapon/tank/plasma/plasmaman(synd_mob), slot_s_store)
+		synd_mob.equip_or_collect(new /obj/item/clothing/mask/breath/(synd_mob), slot_wear_mask)
```
Comment (ComicIronic) [inline code/game/gamemodes/nuclear/nuclear.dm]: Does this set internals automatically, or do you have to use the same as the vox code?

File: code/modules/clothing/spacesuits/plasmamen.dm [hunk 3]
```diff
@@ -76,54 +80,91 @@
...
+	name = "plasmaman atmospheric helmet"
 	icon_state = "plasmamanAtmos_helmet0"
 	base_state = "plasmamanAtmos_helmet"
+	armor = list(melee = 20, bullet = 0, laser = 0,energy = 0, bomb = 25, bio = 100, rad = 0)
+	max_heat_protection_temperature = FIRE_HELMET_MAX_HEAT_PROTECTION_TEMPERATURE
 
 /obj/item/clothing/suit/space/plasmaman/engineer
+	name = "plasmaman engineer suit"
 	icon_state = "plasmamanEngineer_suit"
+	armor = list(melee = 40, bullet = 5, laser = 20,energy = 5, bomb = 35, bio = 100, rad = 80)
```
Comment (ComicIronic) [inline code/modules/clothing/spacesuits/plasmamen.dm]: You also might want to add the pressure_resistance of 200 * ONE_ATMOSPHERE (the standard for engies) to both this and helmet.

File: code/game/gamemodes/nuclear/nuclear.dm (hunk 2 above)
Comment (SkowronX) [inline code/game/gamemodes/nuclear/nuclear.dm]: Missed that. Will add next commit

**Comment by SkowronX** [comment]
All done with that. Time for a few new suits.

**Comment by d3athrow** [comment]
Suggest you make a central proc to handle equipping species equipment to make things nice.

**Comment by SkowronX** [comment]
Ill do that in a separate pr.

**Pull request merged**